net/net/src/TSQLServer.cxx

```diff
@@ -143,6 +143,16 @@ Bool_t TSQLServer::StartTransaction()
    return Exec("START TRANSACTION");
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// returns kTRUE when transaction is running
+/// Must be implemented in derived classes
+
+Bool_t TSQLServer::IsTransaction()
```

Comment:
```suggestion
Bool_t TSQLServer::HasTransactionInFlight()
```
Is that more accurate?

Reply:
Yes, one can use such name, I will modify code